packages/vue-dot/src/elements/PageContainer/PageContainer.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<VSheet
+		class="vd-page-container"
+		:class="paddingClass"
+		:width="containerSize"
+	>
+		<slot />
+	</VSheet>
+</template>
+
+<script lang="ts">
+	import Vue from 'vue';
+	import Component, { mixins } from 'vue-class-component';
+
+	import { IndexedObject } from '../../types';
+
+	const Props = Vue.extend({
+		props: {
+			/**
+			 * Define the size of the container
+			 *
+			 * Default value: "xl"
+			 */
+			size: {
+				type: String,
+				validator: value => [
+					'xl',
+					'l',
+					'm',
+					'xs'
+				].includes(value.toLowerCase()),
+				default: 'xl'
```

Comment:
```suggestion
				type: String,
				default: 'xl',
				validator(value) {
					return ['xl', 'l', 'm', 's'].includes(value.toLowerCase());
				}
```
Pareil c'est juste pour être consistant au niveau de la syntaxe en utilisant `validator(value) {}` et j'ai juste mis les valeurs sur une seule ligne
D'habitude je préfère mettre sur plusieurs lignes mais comme les valeurs sont très courtes je trouve ça plus lisible sur une seule ligne, comme si on lisait une phrase